pytest_doctestplus/plugin.py

```diff
@@ -22,7 +22,7 @@
 
 _pytest_version = Version(pytest.__version__)
 PYTEST_GT_5 = _pytest_version > Version('5.9.9')
-PYTEST_GE_6_3 = _pytest_version >= Version('6.3')
+PYTEST_GE_6_3 = _pytest_version.is_devrelease or _pytest_version >= Version('6.3')
```

Comment:
Or something hacky like this if this doesn't work... 😬 
```python
PYTEST_GT_6_2 = _pytest_version >= Version('6.2.99')
```

Reply:
It works ;), so maybe good enough for now ?